src/utils/fetch.ts

```diff
@@ -0,0 +1,18 @@
+import crossFetch from 'cross-fetch';
+import type { PineconeConfiguration } from '../data';
+
+export const getFetch = (config: PineconeConfiguration) => {
```

Comment:
**question:** Is it worth adding a tiny unit test file for this utility?

Reply:
This is simple enough that I don't think unit-testing would tell us much. I think what would be more valuable here would be to have some next.js-specific integration testing. But I think it's too much additional scope to add as part of this PR. Will have to look into it as a follow-up item.